### ISSUE TITLE
Build environment for Windows?

### ISSUE THREAD

**Issue by emendelson** [kanjitalk755/macemu#161]
The build instructions for Windows say "MinGW32/MSYS2" but it is not clear exactly which version to download. I can build BII and SS on my old system using MSYS32, but that version seems not to be available. I have tried building on the current version of MSYS2, but with an error described here:

https://www.emaculation.com/forum/viewtopic.php?p=76076#p76076

Is there any currently downloadable environment for building BII and SS under Windows?

**Comment by emendelson** [comment]
I read online that current MSYS2 can generate 32-bit executables, but I couldn't find any detailed information about how to do it. I saw a reference to `ARCH=32` so I tried `export ARCH=32` before running autogen.sh, etc., but the same error occurred, and (obviously) I do not know what I am doing.

I can still use my old MSYS32 system to build BII and SS, but anyone who does not already have an old setup will not be able to build on current MSYS, I think. I tried downloading the old MSYS32 installer from archive.org, but the resulting system wasn't able to download anything from the server, because it no longer supports 32-bit MSYS.

I think Cat7 and I may be the only people building on Windows, so this is not an urgent issue.
